psetv2/input.go

```diff
@@ -1148,6 +1151,10 @@ func (i *Input) deserialize(buf *bytes.Buffer) error {
 						return ErrInInvalidBlindedIssuanceValue
 					}
 
+					if i.BlindedIssuance != nil {
+						return ErrInDuplicatedField("blinded issuance flag")
+					}
```

Comment:
in the rest of the code we first check for duplicates,  then for value correctness. Can you please move this before the value validation if block?

Reply:
Done